### DISCUSSION TITLE
Is there a simple human answer to the Encryption warning  message

### DISCUSSION THREAD

**Discussion by thorrrr** [dani-garcia/vaultwarden#3984] [Q&A]
I am sure most of you will say it is easy to understand but for some of us spitting a lot of acronyms for encryption is very confusing.

Do i just pick the new one  or up the old one to 6000001???

![image](https://github.com/dani-garcia/vaultwarden/assets/8170673/895a16f6-58d6-4154-b9fb-ed7070ed2669)

**Comment by stefan0xC** [comment]
> Is there a simple human answer to the Encryption warning message

The simple answer is higher is better (with a second warning that too high might be too much for your devices).

If you click on those question marks the Bitwarden Help page will open with [more info about KDF algorithms](https://bitwarden.com/help/kdf-algorithms/) and [changing KDF iterations](https://bitwarden.com/help/what-encryption-is-used/#changing-kdf-iterations).

> Do i just pick the new one or up the old one to 6000001???

As said in the second warning the recommendation would be to increment the value by steps of `100,000` (with the recommended value being at least `600000`). If you increase it by a factor of 60 in one go your login experience might suffer (i.e. it takes much longer).

You could also use `argon2id` as key derivation function instead of `PBKDF2` (which is the [recommended KDF by OWASP](https://cheatsheetseries.owasp.org/cheatsheets/Password_Storage_Cheat_Sheet.html)).

**Reply by stefan0xC** [reply]
More important than the KDF is your password strength and if you have secured your account with a second factor.
